src/actors/objects/mod.rs

```diff
@@ -550,7 +576,11 @@ impl ObjectsActor {
                             // the search by debug/code id. We do not surface those errors to the
                             // user (instead we default to an empty search result) and only report
                             // them internally.
-                            log::error!("Failed to download from {}: {}", type_name, LogError(&e));
+                            log::error!(
+                                "Source list-files failed for {}: {}",
```

Comment:
_nit_: To be more consistent with other error messages, how about:

```suggestion
                                "Failed to fetch file list from {}: {}",
```